e2e/resources/enums.ts

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+
+export enum workspaceAccessLevel {
+   OWNER = 'OWNER',
+   READER = 'READER',
+   WRITER = 'WRITER',
+}
+
+export enum workspaceAction {
+   DUPLICATE  = 'Duplicate',
+   DELETE = 'Delete',
+   EDIT = 'Edit',
+   SHARE = 'Share',
+}
+
+const configs = require('../resources/workbench-config');
```

Comment:
Is this really plural? Below the code seems to reference properties on it directly, indicating `config` would be a better name